src/Collections/Upstreams.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Pantheon\Terminus\Collections;
+
+use Terminus\Exceptions\TerminusNotFoundException;
+
+class Upstreams extends UserOwnedCollection
```

Comment:
This class should be retrieved through the user, but doesn't actually use the user object anywhere. Should this be a normal `TemrinusCollection`?